src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/cluster.module.ts

```diff
@@ -6,11 +6,11 @@ import { RouterModule } from '@angular/router';
 import { BsDropdownModule } from 'ngx-bootstrap/dropdown';
 import { ModalModule } from 'ngx-bootstrap/modal';
 import { TabsModule } from 'ngx-bootstrap/tabs';
-
```

Comment:
Please re-add this empty line. We separate those includes everywhere across the webui code.